README.md

```diff
@@ -1,17 +1,261 @@
-# Receiving Api Posts
+# Receiving API POSTs
 
 ## Objectives
 
-  * Understand how params comes in from POST requests
-  * Use those params to create resources
+  1. Understand how `params` comes in from POST requests.
+  2. Use those `params` to create resources.
+  3. Return and consume the created resource as JSON.
 
-## Outline
+## Lesson
 
-  * "What if we want to modify the post create form to not require a refresh"
-  * we need to send a POST. Turns out it's not really any different than what the form is doing.
-  * Show them in jQuery how to do the POST
-  * Comes in as the `params`
-  * What should the response be? Well what do we want to show the user after they hit submit? We want to show the post! So let's make the response be just the post that was created
-  * so let's use our AMS post serializer to respond with the post that was created
-  * status code 201
-  * Render the post in jQuery
+So far, we've been focused on enhancing how we read our blog, but what
+about creating blog posts? How can we apply what we've been doing with
+JSON and AJAX to make creating a post a better experience?
+
+### Creating a Post With an AJAX POST
+
+In our `posts/new.html.erb`, we already have a form. If we click the
+button, it will submit as normal and the code in the controller's
+`create` action will do whatever it's set up to do.
+
+What we want to do is set it up so that we use this form, but use jQuery
+and AJAX to submit it, so that we can handle a JSON response and display
+the newly created blog post without redirecting to the post `show` page.
+
+Our first order of business is to prevent the default form submit action and
+do our own thing. To do this, we need to hook up an event handler to the
+form's `submit` event, and then block the form from doing an HTML
+submit as it normally would.
+
+```erb
+# posts/new.html.erb
+<%= form_for(@post) do |f| %>
+  <div class="field">
+    <%= f.label :title %><br>
+    <%= f.text_field :title %>
+  </div>
+  <div class="field">
+    <%= f.label :description %><br>
+    <%= f.text_field :description %>
+  </div>
+  <div class="actions">
+    <%= f.submit %>
+  </div>
+<% end %>
+
+<script type="text/javascript" charset="utf-8">
+  $(function () {
+    $('form').submit(function(event) {
+      //prevent form from submitting the default way
+      event.preventDefault();
+      alert("we r hack3rz);
```

Comment:
I was so caught up in being l33t that I missed the missing quote -- thanks for catching that.